lib/processors/bundlers/moduleBundler.js

```diff
@@ -146,13 +148,20 @@ module.exports = function({resources, options: {bundleDefinition, bundleOptions}
 
 			return Promise.all(bundles.map((bundleObj) => {
 				if ( bundleObj ) {
-					const {name, content} = bundleObj;
+					const {name, content, sourceMap} = bundleObj;
 					// console.log("creating bundle as '%s'", "/resources/" + name);
-					const resource = new EvoResource({
+					const res = {};
+					res.bundle = new EvoResource({
 						path: "/resources/" + name,
 						string: content
 					});
-					return resource;
+					if (sourceMap) {
+						res.sourceMap = new EvoResource({
```

Comment:
What about just pushing all resources (bundle + sourcemap) to the existing array and then picking the files by checking for the resource name / extension?

Right now our tasks don't seem to need a connection between bundle and sourcemap so it could just loop over the array and set the tags according to the file extension. To me this sounds like a feasible solution.

This way we could also stick to the existing API that we intended to be used by all kind of `processors`.

And adopting the existing tests (which I already started..), would be way easier.

Reply:
I discussed this with @codeworrior and as the minifier processor already uses such an API, we will also stick with it here.

Reply:
However, the JSDoc for @returns still needs to be adopted

Reply:
Done